kythe/go/extractors/config/preprocessor/modifier/BUILD

```diff
@@ -0,0 +1,34 @@
+load("//tools:build_rules/shims.bzl", "go_library", "go_test")
+
+package(default_visibility = ["//kythe:default_visibility"])
+
+go_library(
+    name = "modifier",
+    srcs = [
+        "build_gradle_modifier.go",
+        "pom_xml_modifier.go",
+    ],
+    deps = [
+        "@com_github_beevik_etree//:go_default_library",
+    ],
+)
+
+go_test(
```

Comment:
The `go` tool considers all `_test.go` files as part of the same package so the build will break since the same top-level names are used in this test and the one below.

Reply:
ahh damn.  prefer separate packages for both of these then?

Reply:
I'd just make sure the variable names don't clash in the tests.

Reply:
This feels like a way I will shoot myself in the foot again in the future though, so I'm gonna separate them out now.  (Well, again, since that's the way they were before too)

Reply:
You can prevent the foot shooting by combining the Bazel `go_test` targets into one.

Reply:
If we get rid of runextractor binary I may do this then.

For now having separate libraries means we don't have a maven subcommand depending on a bunch of gradle stuff and v. versa.

If the future is combined (which I'm leaning more towards now), I can come back and fix this.